prog/dftb+/lib_dftb/dftd3param.F90

```diff
@@ -469,7 +472,10 @@ pure subroutine copyC6(c6ab)
       &    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp, &
       &    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp, &
       &    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp, &
-      &    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp, &
+      &    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp,    0.0000_dp],&
+      &    [maxRef, maxRef, blockSize])
+
+    c6ab(:, :, 39:57) = reshape([ &
```

Comment:
Not sure if its clearer, but the sizing could be done explicitly with multiples of blockSize in each chunk
```suggestion
    c6ab(:, :, 2*blockSize+1:3*blockSize) = reshape([ &
```